src/main/kotlin/uk/gov/dwp/dataworks/aws/AwsParsing.kt

```diff
@@ -0,0 +1,97 @@
+package uk.gov.dwp.dataworks.aws
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.cfg.MapperConfig
+import com.fasterxml.jackson.databind.introspect.AnnotatedField
+import com.fasterxml.jackson.databind.introspect.AnnotatedMethod
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.io.Resource
+import org.springframework.stereotype.Component
+import uk.gov.dwp.dataworks.AwsIamPolicyJsonObject
+import uk.gov.dwp.dataworks.services.ConfigurationResolver
+import java.lang.reflect.Modifier
+
+@Component
+class AwsParsing(){
```

Comment:
If we are going to pull this out to a separate class (which I approve of) we might consider bringing the resources with the code and remove that bulk from the main `TaskDeploymentService`. This will mean we have a more cohesive class and can remove some tight coupling with the above class regarding those resources

Reply:
The issue that I have with this is that the resources are still going to have to be referenced within the parsePolicyDocument() method call, to make it generic - if you are happy for them to be called as public values within the class of the method, I will change it but I think it is minor and could possibly be more confusing to read.